### PR TITLE
Support MPL in mlkit-bench.

### PR DESCRIPTION
**Pull request by athas**
Unfortunately, the ray tracer is the only program that seems to run at non-glacial speed in MPL.  I have not yet investigated what is going wrong with the others.

### CODE REVIEW

File: par-benchmarks/ray/ray.sml [hunk 1]
```diff
@@ -150,10 +150,7 @@ fun mk_bvh f all_objs =
                 val xs_right = List.drop(xs_sorted, n div 2)
                 fun do_left () = mk (d+1) (n div 2) xs_left
                 fun do_right () = mk (d+1) (n-(n div 2)) xs_right
-                val (left, right) =
-                    if n < 200
-                    then (do_left(), do_right())
-                    else ForkJoinSeq.par (do_left, do_right)
+                val (left, right) = (do_left(), do_right())
```
Comment (melsman) [inline par-benchmarks/ray/ray.sml:153]: Nice catch...

File: par-benchmarks/util/ForkJoinMPL.sml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+val par = FJ.par
+val parfor = FJ.parfor
+val pmap = List.map
+
+fun alloc n _ = Array.alloc n
+fun pair (f,g) (x,y) = par (fn () => f x, fn () => g y)
+
+type gcs = int * int
+fun parfor' _ = parfor 0
```
Comment (melsman) [inline par-benchmarks/util/ForkJoinMPL.sml]: We could write

    fun parfor' ((_,g):gcs) = parfor g

Comment (athas) [reply par-benchmarks/util/ForkJoinMPL.sml]: Oh yeah, that's probably better, I'll change it...

Comment (athas) [reply par-benchmarks/util/ForkJoinMPL.sml]: This was also the reason why MPL wouldn't run most of the bencmarks.  Much better now!

**Comment by melsman** [comment]
Great!

**Pull request merged**